lisa_commands/commands.py

```diff
@@ -136,17 +202,97 @@ def fetch_image(query):
     # If the user added 'from google' or 'from bing' to end of command
     # use that engine.
     if len(split) > 1 and split[1].lower() == 'from':
+        logging.warning('IN FROM')
         text = None
         if split[2].lower() == 'google':
             text = request.google_image_search(query)
         elif split[2].lower() == 'bing':
             text = request.bing_image_search(query)
 
         if text is None: text = 'Sorry no results found.'
+
         return text
     # If the user didn't specify a search engine we just pick one
     search_engine = random.choice(['google', 'bing'])
     if search_engine == 'google':
         return "From Google: " + request.google_image_search(query)
-
     return "From Bing: " + request.bing_image_search(query)
+
+def kill_me(query, _slack_event):
+    """
+    query: query str
+    slack_event: A dict of slack event information(unused for this function)
+    returns the text to send back to slack
+    """
+    return request.gify_search('gifs', 'kill me')
+
+def manga_me(query, _slack_event):
+    """
+    query: query str
+    slack_event: A dict of slack event information(unused for this function)
+    returns the text to send back to slack
+    """
+    return request.anime_news_network_search('manga', query)
+
+def put_it_back(_query, _slack_event):
+    """
+    query: query str(unused for this function)
+    slack_event: A dict of slack event information(unused for this function)
+    returns the text to send back to slack
+    """
+    return random.choice(["┬─┬ノ( º _ ºノ)", r"┬──┬ ¯\_(ツ)"])
+
+def reverse_me(query, _slack_event):
+    """
+    query: query str
+    slack_event: A dict of slack event information(unused for this function)
+    returns the text to send back to slack
+    """
+    return query[::-1]
+
+def shame(query, _slack_event):
+    """
+    query: query str
+    slack_event: A dict of slack event information(unused for this function)
+    returns the text to send back to slack
+    """
+    user = query.strip().upper()
+    return random.choice([
+        'Shame on you! ' + user + 'You should know better!',
+        user + ' ಠ_ಠ',
+        user + ' You have made a mockery of yourself. Turn in your weeabo credentials!',
+        user + ' :blobdisapproval:',
+        user + ' :disappoint:',
+        user + ' you did bad and you should feel bad',
+        user + ' :smh:',
+    ])
+def sticker_me(query, _slack_event):
```

Comment:
As we discussed, no return on this bad boi.

Reply:
Fixed!

Reply:
Added!